_posts/2019-10-25-Kickstarter-Trajectory-2019-Summer-Edition.md

```diff
@@ -0,0 +1,219 @@
+---
+layout: post
+authors: [brecht_porrez, duncan_casteleyn, imad_hamroun, jasper_rosiers, lennert_peeters, lore_vanderlinden, mohammed_laghzaoui, nicholas_meyers]
+title: "Kickstarter Trajectory 2019 Summer Edition"
+image: /img/kicks.png
+tags: [Spring, Spring Boot, Angular, Unit testing, Mocking, Microservices, Git, DevOps, Docker, TypeScript, Kickstarter, Security]
+category: Kickstarters
+comments: true
+---
+
+# Introduction
+>65 young professionals started the Ordina kickstarter trajectory this summer, on the 1st of August. 
+JWorks counted 8 kickstarters: Mohammed, Jasper, Nicholas, Lennert, Duncan, Lore, Brecht and Imad. 
+All looking for a challenge and a fresh start. Most of them just graduated, Brecht on the other hand already had other work experience. 
+During the Kickstarter program, we were introduced to a wide range of technologies, technologies that play an important role in IT. 
+The courses were focused mainly on providing a very hands-on experience as to learn how the different technologies work in practice.
+
+During the 3 months kickstarter trajectory, there were 2 main periods:
+* The first six weeks were filled with various trainings: security, backend, frontend, soft skill trainings,...
+* The last, 2 DevCases were organized: Chatbot Dina and Zero Plastic Rivers
+
+This blogpost will talk about both periods separately below. The first period is further separated into technical and non-technical trainings.
+
+1.[Technical trainings](#technical-trainings)  
+2.[Non-technical trainings](#non-technical-trainings)  
+3.[DevCases](#devcases) 
+
+## First Day
+On our first arrival at Ordina, we were warmly welcomed by Anja and the Jworks team. 
+We were introduced to the structure of Ordina and got to know some of the other Kickstarters. 
+After the introduction, we were given a tour of the entire building, during which we met some of the colleagues at JWorks.
+Four of the Kickstarters already did an internship at Ordina before coming to work officially, so they already knew a lot of the colleagues in Mechelen.
+This didn't make any difference however, since everyone was very open and welcoming.
+Finally, we were given the keys to our cars, and our laptops, so that the next day we could get started with the courses, focused on both technical and soft skills.
```

Comment:
Maybe rewrite the sentence as the overuse of commas (4) breaks the flow a lot. It also reads as if you got the keys to your laptops...
So maybe something like: 

> Finally, we got our laptops and the keys to our cars so that the next day we could get started with the courses which were focused on both technical and soft skills.